azurerm/internal/services/compute/linux_virtual_machine_scale_set_resource.go

```diff
@@ -251,6 +251,11 @@ func resourceArmLinuxVirtualMachineScaleSet() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+
+			"encryption_at_host": {
+				Type:     schema.TypeBool,
+				Optional: true,
+			},
```

Comment:
this field isn't computed - can we move it up inline (sorted alphabetically) within the "optional" fields above?

can we also add this to the documentation? (`./website/docs/r/{os}_virtual_machine_scale_set.markdown`)